doc/release-30635.md

```diff
@@ -1,4 +1,5 @@
 Updated RPCs
 ------------
 
-- The waitfornewblock RPC takes an optional `current_tip` argument. (#30635)
+- The waitfornewblock now takes an optional `current_tip` argument. It is also no longer hidden. (#30635)
```

Comment:
In commit "rpc: unhide waitfor{block,newblock,blockheight}" (c6e2c31c55123cc97b4400bcbf3c37a39b067a22)

Accidentally dropped the word RPC?

Reply:
It seems so. Will fix release note if I have to retouch.